README.md

```diff
@@ -1,13 +1,19 @@
 # Uptane Standard
 
-[Uptane](https://uptane.github.io) is the first compromise-resilient software update security system for the automotive industry. Beginning in 2018, a working group has begun the process of describing the system's design, implementation, and deployment considerations as a formal standard, working as an [IEEE-ISTO](https://ieee-isto.org/) industry group and is currently a [Linux Foundation Joint Development Foundation](http://www.jointdevelopment.org/) project. This repository is the public home of that standardization work.
+[Uptane](https://uptane.github.io) is the first compromise-resilient software update security system for the automotive industry. In 2018, a working group under [IEEE-ISTO](https://ieee-isto.org/) began the process of describing the system's design, implementation, and deployment considerations as a formal standard. On July 31, IEEE/ISTO released *IEEE-ISTO 6100.1.0.0: Uptane Standard for Design and Implementation* (see link below under documentation). Uptane is now a [Linux Foundation Joint Development Foundation](http://www.jointdevelopment.org/) project. 
+
+This repository is the public home of all standardization work for Uptane.
 
 ## Existing documentation
 
-Several documents are already available which describe Uptane's design and implementation. Note that an intial version of the Uptane Standards document has been developed and should be considered the authoritative resource for the framework:
+The Uptane Standards document should be considered the authoritative resource for the framework. Several other documents and materials are available or currently in development. The information in all of these other guidelines should be viewed as complementary to the official Uptane standard, and as recommendations rather than mandatory instructions. 
 
 * [Uptane Standards Document](https://uptane.github.io/uptane-standard/uptane-standard.html)
```

Comment:
Sorry, @jhdalek55, but this is exactly what I meant yesterday. Do we point to IEEE-ISTO standard, or the unstable / unversioned one?

Reply:
I'm happy to leave this decision making to others. Can I ask, though why you think we should be pointing to the unversioned one? I'm not being confrontational here. I'm just trying to understand the rationale. Is it so the user can have access to the most up-to-the minute version? Or to encourage input? Given that any versioned edition has the advantage of multiple reviews, isn't it a more secure choice to offer to people? 

Is this something the group as a whole should discuss? I'm working on the agenda now and can add it as an item.

At any rate, can you check the rest of the document and if its OK, can I get the rest of the text merged?  It would be a simple enough things to do a separate PR to fix the number. Its my experience lately that these pull requests become quite ungainly because debates on one small point end up preventing the updating of other pieces. I have had to re-do two pull requests lately that have become swollen with multiple added commits. The text in the opening paragraph needs to be fixed because it does not mention the release of the standard and it looks like we have not completed that text. I really would like to get that fixed.

Lois